pkg/mgrconfig/config.go

```diff
@@ -114,4 +114,8 @@ type Config struct {
 	SyzFuzzerBin   string `json:"-"`
 	SyzExecprogBin string `json:"-"`
 	SyzExecutorBin string `json:"-"`
+
+	// MAB task selection and seed selection
+	MABTS bool `json:"mabts"`
```

Comment:
This is not enabled. How do users are expected to make the choice of enabling it? Enabling only one, or the other, or both? Do we encourage disabling both? In what cases? It all needs to be documented if we are giving users choice.
Note that lots of users may be not qualified to do such choices and in almost 100% of cases are not interested in spending time on fine tuning such things and if it's not enabled by default, it does not bring value to most users.
Can we do withing adding any flags?